libraries/Microsoft.Bot.Builder/Teams/TeamsActivityExtensions.cs

```diff
@@ -6,20 +6,38 @@
 
 namespace Microsoft.Bot.Builder.Teams
 {
+    /// <summary>
+    /// The TeamsActivityExtensions
+    /// provides helpers to make interacting with Microsoft Teams objects easier. 
+    /// </summary>
     public static class TeamsActivityExtensions
     {
+        /// <summary>
+        /// Gets the Team's channel id off the current activity.
```

Comment:
Should it be: "Gets the Teams' channel Id from the current activity"?

Reply:
updated